src/main/kotlin/no/nav/tilleggsstonader/sak/opplysninger/søknad/domain/SøknadBarn.kt

```diff
@@ -18,5 +20,21 @@ data class SøknadBarn(
 data class BarnMedBarnepass(
     val type: TypeBarnepass,
     val startetIFemte: JaNei?,
+    val utgifter: Utgifter?,
     val årsak: ÅrsakBarnepass?,
 )
+
+data class Utgifter(
+    val harUtgifterTilPassHelePerioden: JaNei,
+    val fom: LocalDate?,
+    val tom: LocalDate?,
+)
+
+fun UtgifterKontrakter?.tilDomene(): Utgifter? =
+    this?.let {
+        Utgifter(
+            harUtgifterTilPassHelePerioden = it.harUtgifterTilPass.verdi,
```

Comment:
Hadde vært fint å oppdatere kontraktet sånn at feltet der også blir litt tydligere før vi tar denne i bruk :) 
```suggestion
            harUtgifterTilPassHelePerioden = it.harUtgifterTilPassHelePerioden.verdi,
```

Reply:
Fikset